WrightTools/artists.py

```diff
@@ -154,6 +338,117 @@ def legend(self, *args, **kwargs):
             kwargs['framealpha'] = 1.
         return super().legend(*args, **kwargs)
 
+    def pcolor(self, *args, **kwargs):
+        """Create a pseudocolor plot of a 2-D array.
+
+        Parameters
+        ----------
+        data : 2D WrightTools.data.Data object
+            Data to plot.
+        channel : int or string (optional)
+            Channel index or name. Default is 0.
+        dynamic_range : boolean (optional)
+            Force plotting of all contours, overloading for major extent. Only applies to signed
+            data. Default is False.
+        label : {'none', 'both', 'x', 'y'}  (optional)
+            Parameterize application of labels directly from data object. Default is none.
+        xlabel : string (optional)
+            xlabel. Default is None.
+        ylabel : string (optional)
+            ylabel. Default is None.
+        **kwargs
+            matplotlib.axes.Axes.pcolor__ optional keyword arguments.
+
+        __ https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.pcolor.html
+
+        Returns
+        -------
+        QuadContourSet
```

Comment:
Pretty sure pcolor doesn't return QuadContourSet

On all of these, they should be `matplotlib.collections.Collection` or similar to enable intersphinx

Reply:
great catch